fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBStoreTimer.java

```diff
@@ -178,14 +179,27 @@
         TIME_WINDOW_LEADERBOARD_SAVE_SUB_DIRECTORY("leaderboard save sub-directory");
 
         private final String title;
+        private final String logKey;
         Events(String title) {
             this.title = title;
+            this.logKey = this.name();
+        }
+
+        Events(String title, String logKey) {
+            this.title = title;
+            this.logKey = logKey;
```

Comment:
Similar comment about constructors. I won't comment on any of the other constructors in this PR...you get the idea.